### PR TITLE
Sidebar categories are no longer covered by header

### PR DESCRIPTION
**Pull request by celestinebuendia**
- Shortened sidebar to no longer overlap with header, so that course categories rendered at the top or sidebar are not covered
- Made the corresponding adjustment to have the header flush with bottom of screen

Testing
- Sidebar still adjusts proportionally to window resizing
- Saw that sidebar buttons are no longer hidden by header

### CODE REVIEW

**Review by jeremynguyen593 — APPROVED** [review]

File: app/src/Homepage/Homepage.css [hunk 1]
```diff
@@ -15,8 +15,9 @@
 /* Sidebar css */
 .sidebar {
   background-color: #F0F0F0;
-  height: 100%;
+  height: 89%;
   width: 15%;
+  bottom: 0%;
```
Comment (jeremynguyen593) [inline app/src/Homepage/Homepage.css:20]: Is this bottom: 0% needed?

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

File: app/src/Homepage/Homepage.css [hunk 2]
```diff
@@ -15,8 +15,9 @@
 /* Sidebar css */
 .sidebar {
   background-color: #F0F0F0;
```
Comment (jeremynguyen593) [inline app/src/Homepage/Homepage.css:17]: This isn't you but can you change these to lowercase to follow the style guide?

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

File: app/src/Homepage/Homepage.css (hunk 2 above)
Comment (celestinebuendia) [reply app/src/Homepage/Homepage.css:17]: Sure

File: app/src/Homepage/Homepage.css (hunk 1 above)
Comment (celestinebuendia) [reply app/src/Homepage/Homepage.css:20]: Yes, without setting the bottom field to 0, the sidebar will begin rendering at the top of the page and will still be covered by the header. 

There may be some other way to do this (I could also move the buttons down) but my method was to shorten the sidebar and make it flush to the bottom of the page.

**Review by jeremynguyen593 — APPROVED** [review]

**Pull request merged**